lib/dfe/analytics/fields.rb

```diff
@@ -71,23 +62,6 @@ def self.hidden_pii
         DfE::Analytics.hidden_pii || {}
       end
 
-      def self.allowlist_pii
```

Comment:
@ericaporter do we still have the concept of requiring each field in each table to be in *either* analytics_blocklist.yml or analytics.yml, plus optionally analytics_hidden_pii.yml? Enforcement of that first 'or' needs to remain.

I think you probably haven't changed that here but wanted to double check!

Reply:
We've got the originals validations still which are:
1) fields appearing in analytics.yml that have been removed
2) new fields detected that aren't in analytics.yml or blocklist.yml
3) fields existing in both analytics & blocklist